.github/CHANGELOG.md

```diff
@@ -49,6 +49,9 @@
 * Add `isort` to `requirements-dev.txt` and run before `black` upon `make format` to sort Python imports.
   [(#623)](https://github.com/PennyLaneAI/pennylane-lightning/pull/623)
 
+* Improve support for new operator arithmetic with `QuantumScriptSerializer.serialize_observables`.
```

Comment:
The helper function `QuantumScriptSerializer._ob` is primarily used by `serialize_observables` (and recursive calls but those don't matter in this context). Since `serialize_observables` is the public function, I put that in the changelog. Let me know if you think I should change this.